tests/external_botocore/test_bedrock_chat_completion.py

```diff
@@ -255,6 +268,53 @@ def _test():
     _test()
 
 
+@pytest.mark.parametrize(
+    "llm_token_callback",
+    [
+        llm_token_count_callback_success,
```

Comment:
Given that we are now extensively testing the common implementation in agent features we can reduce the tests here to with and without the token counting callback. Note we already have tests without the callback so we just need to add tests with the callback returning a valid count.

Reply:
Note we _do_ need to test running with the callback during error conditions to make sure we are still capturing token counts where we have the content during error conditions.